rest_api/rest_api/pipeline/__init__.py

```diff
@@ -25,12 +32,70 @@ def setup_pipelines() -> Dict[str, Any]:
     pipelines = {}
 
     # Load query pipeline
-    query_pipeline = Pipeline.load_from_yaml(Path(config.PIPELINE_YAML_PATH), pipeline_name=config.QUERY_PIPELINE_NAME)
+    # query_pipeline = Pipeline.load_from_yaml(Path(config.PIPELINE_YAML_PATH), pipeline_name=config.QUERY_PIPELINE_NAME)
+
+    # ------------------
+    document_store = ElasticsearchDocumentStore(host="host.docker.internal")
+    extractive_document_store = ElasticsearchDocumentStore(
+        host="host.docker.internal", index="rulebook", embedding_dim=768
```

Comment:
Hey Rafal, I see you have set embedding_dim on the extractive document store. Did you try using an EmbeddingRetriever for the Extractive QA Pipeline as embedding_dims are not used by the BM25 Retriever, if so was the BM25 reader better in your view?

Reply:
I've set embedding_dim here to 768, but it's also the default of any DocumentStore in haystack, left it here for no particular reason tbh. Regarding the different retrievers - I didn't experiment with different architectures, used BM25, as in literature that's the best model (combination of BM25 + Dense reranking model), but for sure we can experiment later on with different sets of retrievers for extractive part